addons/tmf_loadouts/loadouts/ussr_army_1985.hpp

```diff
@@ -384,18 +384,22 @@ class sp : g {
 	};
 };
 
-class vg : smg {
+class vg : car {
 	displayName = "Vehicle Gunner";
-	vest[] = {"CUP_V_CDF_6B3_2_Green"};
-	headgear[] = {"CUP_H_TK_TankerHelmet"};
+	uniform[] = {"SP_0000_Standard_BattleDressUniform_Black"};
+	vest[] = {"CUP_V_CDF_OfficerBelt"};
+	headgear[] = {"CUP_H_SLA_TankerHelmet"};
+	sidearmWeapon[] = {"CUP_hgun_Makarov"};
```

Comment:
Either give them a primary or a pistol, alternatively give the commander a primary.